.circleci/config.yml

```
@@ -122,8 +122,9 @@ commands:
           name: syntax
           command: |
             pip install "pycodestyle>=2.10.0,<3.0.0" ruff
-            if [ -n "${RUN_SYNTAX}" ]; then pycodestyle synapse; ruff check; fi;
-            if [ -n "${RUN_SYNTAX}" ]; then pycodestyle scripts; ruff check; fi;
+            if [ -n "${RUN_SYNTAX}" ]; ruff check; then pycodestyle synapse; fi;
+            if [ -n "${RUN_SYNTAX}" ]; ruff check; then pycodestyle scripts; fi;
+            if [ -n "${RUN_SYNTAX}" ]; ruff check; then pycodestyle examples; fi;
```

Can this all be combined?

```suggestion
            if [ -n "${RUN_SYNTAX}" ]; then
                ruff check
                pycodestyle synapse
                pycodestyle scripts
                pycodestyle examples
            fi
```